libs/jacdac-services/servoclient.ts

```diff
@@ -17,11 +36,12 @@ namespace jacdac {
         //% servo.fieldOptions.columns=2
         //% blockGap=8        
         setAngle(degrees: number) {
-            if (!this.state[0] || this.state.getNumber(NumberFormat.Int16LE, 1) != degrees) {
-                this.state[0] = 1;
-                this.state.setNumber(NumberFormat.Int16LE, 1, degrees);
-                this.notifyChange();
-            }
+            // this isn't exactly what the internets say, but it's what codal does
+            const center = 1500
```

Comment:
it's not uncommon that these values need to be tuned based on the drift of the servo.

Reply:
I think I'll change the service API to have these values as registers, and the value be the angle (probably 0-0xffff); then it can come pre-configured by the jd-servo manufacturer but be changed as needed

Reply:
If there is drift over usage, then I think compensation will have to be stored on the brain, not in the peripheral. Peripheral will be stateless.